src/client/pages/Geo/GeoMap/GeoMap.tsx

```diff
@@ -35,4 +70,11 @@ const GeoMap: React.FC<Props> = ({ children, center, zoom }) => {
   )
 }
 
+GeoMap.defaultProps = {
+  // Default to Finland's centroid and bounds
```

Comment:
I don't think there should a default here. user always lands on a country page, therefore countryIso is there.

Reply:
Done 👍